src/sql/workbench/contrib/dashboard/browser/contents/controlHostContent.component.ts

```diff
@@ -71,6 +74,8 @@ export class ControlHostContent {
 	}
 
 	public refresh() {
-		this._agentViewComponent.refresh = true;
+		if (this._agentViewComponent !== undefined) {
+			this._agentViewComponent.refresh = true;
```

Comment:
Why is this a property and not a method?

Reply:
@Charles-Gagnon agentViewComponent is not an sql assessment part. I don't know why the agent extension author did this as a property.

Reply:
Ah, misread that. Ok yeah it's fine as is.

Don't you want to handle refresh for your component though?

Reply:
We will do this in a further release.